services/challengedeployerservice/controller.go

```diff
@@ -18,11 +17,75 @@ import (
 	"github.com/sdslabs/katana/lib/utils"
 	"github.com/sdslabs/katana/types"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+    _ "unsafe"
 )
+//for get request
+func Deploy(c *fiber.Ctx) error {
+	patch := false
+	replicas := int32(1)
+	log.Println("Starting")
+	//read folder challenge by os
```

Comment:
Keep consistency in your comments, some are starting with lower case while others are using upper case, look at the code base and find out how it is done

Reply:
Done